src/services/FetchMovies.php

```diff
@@ -6,18 +6,22 @@
 
 class FetchMovies
 {
+    const BASE_URL = "https://api.themoviedb.org/3";
     protected $api_key = "f312ac2cb63002f508d52fd432cea28d";
     protected $nowPlaying_url = "https://api.themoviedb.org/3/movie/latest?api_key=";
+    protected $upcoming_movies_url = "https://api.themoviedb.org/3/discover/movie?api_key=";
```

Comment:
Please confirm we still need all the class properties. Remove what we no longer use.